frontend/src/App.svelte

```diff
@@ -60,7 +48,7 @@
       "italian food, cezanne painting",
     ],
   };
-  let value: string = promptOptionsByImage["valley"][0];
+  let value: string = "Enter prompt here";
```

Comment:
Do we want to make this placeholder text instead? Updating the value seems odd
![CleanShot 2024-12-17 at 15 05 58@2x](https://github.com/user-attachments/assets/46bcf7cd-e4e8-41c1-89e8-622fef296749)

Reply:
Yeah you're def correct here.